tests/shared/UserLogonStateChangeTestDefs.js

```diff
@@ -24,45 +24,48 @@ fluid.require("%universal");
 
 gpii.loadTestingSupport();
 
-fluid.defaults("gpii.tests.userLogonStateChange.logonChange", {
+fluid.defaults("gpii.tests.userLogonHandling.proximityTriggered", {
     gradeNames: "kettle.test.request.http",
-    path: "/user/%userToken/logonChange",
+    path: "/user/%userToken/proximityTriggered",
     termMap: {
         userToken: "{tests}.options.userToken"
     }
 });
 
-fluid.defaults("gpii.tests.userLogonStateChange.testCaseHolder", {
-    gradeNames: "gpii.test.common.testCaseHolder",
+fluid.defaults("gpii.tests.userLogonHandling.testCaseHolder", {
```

Comment:
I kept the name relatively generic, since the the last couple of tests in this file are for the /login and /logout endpoints